tests/system/base.py

```diff
@@ -73,6 +74,17 @@ def setUp(self) -> None:
         self.vote_service.clear_all()
         self.anon_client = self.create_client()
 
+    def tearDown(self) -> None:
```

Comment:
This can then be removed as well if we somehow disable this behaviour, I think

Reply:
Let's talk about it!

Reply:
see above

Reply:
This can't be removed, because the action_worker-thread is always used